### PR TITLE
[FIX] Making sure the 'hide agent info' hides the agent info even with department change.

### PR DESCRIPTION
**Pull request by cauefcr**
Title says it all.

### CODE REVIEW

File: src/components/helpers.js [hunk 1]
```diff
@@ -155,6 +155,10 @@ export const normalizeTransferHistoryMessage = (transferData, sender) => {
 		},
 		department: () => {
 			const to = nextDepartment && nextDepartment.name;
+			if (!sender.username) {
+				const from = 'the agent';
```
Comment (murtaza98) [inline src/components/helpers.js:159]: We'd have to handle translations for this string. I'd suggest adding a new translation string altogether, in-case agent info is hidden

File: src/components/helpers.js [hunk 2]
```diff
@@ -155,6 +155,9 @@ export const normalizeTransferHistoryMessage = (transferData, sender) => {
 		},
 		department: () => {
 			const to = nextDepartment && nextDepartment.name;
+			if (!sender.username) {
+				return I18n.t('the agent transferred the chat to the department %{to}', { to });
```
Comment (KevLehman) [inline src/components/helpers.js:159]:
```suggestion
				return I18n.t('The agent transferred the chat to the department %{to}', { to });
```

A bit nitpicky but 🙈

Comment (MartinSchoeler) [reply src/components/helpers.js:159]: Wont this conflict with the key? `the_agent_transferred_the_chat_to_the_department_t_3cbe495c` seems to not have caps

Comment (cauefcr) [reply src/components/helpers.js]: changes done

**Review by MartinSchoeler — APPROVED** [review]

**Review by KevLehman — APPROVED** [review]
Code-wise looks good. Pls try before merging 👀

**Pull request merged**